package/Resources/Translations/tr/Translation.ini

```diff
@@ -0,0 +1,1735 @@
+[General]
+Author=LychALPHA
+MapEncoding=utf-8

```

Comment:
You can just remove this line `MapEncoding=utf-8` since it's the default option.

`MapEncoding` key is only used for a special case, e.g. tiberian sun.